src/main/java/net/mcreator/element/GeneratableElement.java

```diff
@@ -42,7 +42,7 @@
 
 public abstract class GeneratableElement {
 
-	public static final int formatVersion = 56;
+	public static final int formatVersion = 57;

```

Comment:
Master version is already at 57 so this PR should update(merge) master and bump to 58

Reply:
Fair. I guess I stopped noticing the version changes.

Reply:
✔️ Fixed